DTS-STN/Service-Canada-Labs
# Storybook Entries

# [[Storybook Entries](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=162910)]

- Upgraded to Storybook 8
- Added storybook entries for Fragment Renderer Components
- Updated next/image imports
- Implemented next/jest and removed babel-jest

## Test Instructions
1. yarn storybook
2. Navigate through the Storybook entries for fragment renderer and confirm they are working properly.
